src/GeoFormatTypes.jl

```diff
@@ -37,6 +37,7 @@ Trait specifying that a format object, like [`WellKnownText`](@ref),
 contains geometry data.
 """
 struct Geom <: FormatMode end
+Base.show(io::IO, ::Type{Geom}) = print(io, "Geometry mode")
```

Comment:
Should these all have mime types?

```suggestion
Base.show(io::IO, ::MIME"text/plain", ::Type{Geom}) = print(io, "Geometry mode")
```

Reply:
Why be selective though? Mainly useful for multiline output, but were not tables or anything, just short statements?